packages/picasso.js/src/core/chart-components/labels/labels.js

```diff
@@ -2,15 +2,20 @@ import strategies from './strategies';
 
 /**
  * @typedef {object} ComponentLabels
- * @property {string} [type='labels']
+ * @extends ComponentSettings
  */
 
 /**
+ * Component settings
  * @typedef {object} ComponentLabels.settings
```

Comment:
```suggestion
 * @typedef {object} ComponentLabels~Settings
```
to be more consistent with the changes on other components

Reply:
Yeah so I would like to use `~` only when we have multiple different settings objects. Like in the case of the axis component. If there is only a single setting it should use `.` IMO. I'll update it.

Reply:
I've updated the spec to use `.` when possible and `~` only for special cases